src/sync.cpp

```diff
@@ -732,6 +732,9 @@ namespace librealsense
         if (!composite)
         {
             std::vector<frame_holder> match;
+            std::stringstream error_message_holder;
+            error_message_holder << f; // Saving frame holder string before moving frame
```

Comment:
Please rename to `frame_string_for_logging`

Reply:
I assume it is done?
@Tamir91 when you update according to a comment please add a "Done" comment

Reply:
> I assume it is done? @Tamir91 when you update according to a comment please add a "Done" comment

Eyes, I talked offline with Ohad about changes...